plugins/mgm/src/main/kotlin/net/corda/cli/plugins/mgm/GenerateGroupPolicy.kt

```diff
@@ -63,109 +64,76 @@ class GenerateGroupPolicy(private val output: GroupPolicyOutput = ConsoleGroupPo
      * Creates the content of the GroupPolicy JSON.
      */
     private fun generateGroupPolicyContent(): Map<String, Any> {
-        val groupPolicy = mutableMapOf<String, Any>()
-        groupPolicy["fileFormatVersion"] = 1
-        groupPolicy["groupId"] = "ABC123"
-        groupPolicy["registrationProtocol"] = "net.corda.membership.impl.registration.staticnetwork.StaticMemberRegistrationService"
-        groupPolicy["synchronisationProtocolFactory"] = "net.corda.v5.mgm.MGMSynchronisationProtocolFactory"
-        groupPolicy["protocolParameters"] = mutableMapOf(
-            "identityTrustStore" to listOf(
-                "-----BEGIN CERTIFICATE-----\nMIICCDCJDBZFSiI=\n-----END CERTIFICATE-----\n",
-                "-----BEGIN CERTIFICATE-----\nMIIFPDCzIlifT20M\n-----END CERTIFICATE-----"
-            ),
-            "tlsTrustStore" to listOf(
-                "-----BEGIN CERTIFICATE-----\nMIIDxTCCE6N36B9K\n-----END CERTIFICATE-----\n",
-                "-----BEGIN CERTIFICATE-----\nMIIDdTCCKSZp4A==\n-----END CERTIFICATE-----",
-                "-----BEGIN CERTIFICATE-----\nMIIFPDCCIlifT20M\n-----END CERTIFICATE-----"
+        return mapOf(
+            "fileFormatVersion" to 1,
+            "groupId" to UUID.randomUUID().toString(),
+            "registrationProtocol" to "net.corda.membership.impl.registration.staticnetwork.StaticMemberRegistrationService",
+            "synchronisationProtocol" to "net.corda.membership.impl.sync.staticnetwork.StaticMemberSyncService",
+            "protocolParameters" to mapOf(
+                "sessionKeyPolicy" to "Combined",
+                "staticNetwork" to mapOf(
+                    "members" to members,
+                )
             ),
-            "tlsPki" to "C5",
-            "p2pProtocolMode" to "AUTHENTICATED_ENCRYPTION",
-            "mgmInfo" to mapOf(
-                "name" to "C=GB, L=London, O=Corda Network, OU=MGM, CN=Corda Network MGM",
-                "sessionKey" to "-----BEGIN PUBLIC KEY-----\nMFkwEwYHK+B3YGgcIALw==\n-----END PUBLIC KEY-----\n",
-                "certificate" to listOf(
-                    "-----BEGIN CERTIFICATE-----\nMIICxjCCRG11cu1\n-----END CERTIFICATE-----\n",
-                    "-----BEGIN CERTIFICATE-----\nMIIB/TCCDJOIjhJ\n-----END CERTIFICATE-----\n",
-                    "-----BEGIN CERTIFICATE-----\nMIICCDCCDZFSiI=\n-----END CERTIFICATE-----\n",
-                    "-----BEGIN CERTIFICATE-----\nMIIFPDCClifT20M\n-----END CERTIFICATE-----"
-                ),
-                "ecdhKey" to "-----BEGIN PUBLIC KEY-----\nMCowBQYDH8Tc=\n-----END PUBLIC KEY-----\n",
-                "keys" to listOf(
-                    "-----BEGIN PUBLIC KEY-----\nMFkwEwYHgcIALw==\n-----END PUBLIC KEY-----\n"
+            "p2pParameters" to mapOf(
+                "sessionTrustRoots" to listOf(
+                    "-----BEGIN CERTIFICATE-----\n{truncated for readability}\n-----END CERTIFICATE-----",
```

Comment:
`Appendix B: Sample Group Policy for Static Group without truncated data` from [group-policy.md](https://github.com/corda/platform-eng-design/blob/master/core/corda-5/corda-5.0/the-host/group-policy.md) has a section with real certs instead of the placeholder `{truncated for readability}`. Could be better to use a more realistic sample here.